client/MentorPanel/src/components/scrolling_questions.js

```diff
@@ -6,29 +6,49 @@ import { withStyles } from '@material-ui/core/styles'
 
 import { normalizeString } from 'src/funcs/funcs'
 
-const ScrollingQuestions = ({ questions, questions_asked, onQuestionSelected, ...props }) => {
+const ScrollingQuestions = ({ questions, questions_asked, recommended, onQuestionSelected, ...props }) => {
   const { classes } = props;
 
   useEffect(() => {
     const top_question = questions.find(q => {
       return !questions_asked.includes(normalizeString(q))
     })
-
-    const node = document.getElementById(top_question);
+    const node = document.getElementById(top_question)
+    if (!top_question || !node) {
+      return
+    }
     node.scrollIntoView({
       behavior: 'smooth',
       block: 'start',
     })
   })
 
+  questions.sort((a, b) => {
+    if (recommended.includes(a) && recommended.includes(b)) {
+      return 0
```

Comment:
this looks suspicious to me. It basically says "any two questions that appear in both lists are equal"

Is there a more explict way the recommended questions are sorted?

It might be this, no?

```
if (recommended.includes(a) && recommended.includes(b)) {
    return questions.indexOf(a) - questions.indeOf(b)
}
```